sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/NettyAsyncHttpClientBuilder.java

```diff
@@ -63,10 +63,9 @@ public NettyAsyncHttpClient build() {
                             nettyProxy = ProxyProvider.Proxy.SOCKS5;
                             break;
                         default:
-                            throw logger.logExceptionAsWarning(
-                                new IllegalStateException(
-                                    "Unknown Proxy type '" + proxyOptions.getType()
-                                        + "' in use. Not configuring Netty proxy."));
+                            throw logger.logExceptionAsWarning(new IllegalStateException(
```

Comment:
Is this going to not use Proxy or will this result in failing to create a HttpClient? If we fail to create the HttpClient, it should be logged as error. If we create a client without proxy, then warning is fine.

Reply:
Good catch